careers.html

```diff
@@ -0,0 +1,143 @@
+        </div>
+    </div>
+    <div class="row margin-top-10">
+        <div class="col l6">
+            <p class="life-info-heading">Culture</p>
+            <p class="life-info">It takes an entire team united behind something big. Together, we work hard, we laugh a lot, we brainstorm nonstop, we use hundreds of Post-Its a week, and we give the best high-fives in town. Also we offer work from home to ensure the comfortess of our team members.</p>
+        </div>
+        <div class="col l6">
+            <img src="img/teamgroup.png">
+        </div>
+    </div>
+</section>
+
+<section class="join-us" id="js-join-us">
+    <p class="section-heading">Join us</p>
+    <p class="heading-bottom-line"></p>
+    <p class="section-subline">If you like getting things done, Infratab is the place for you. We are always looking for people who want to make a difference with their work.</p>
+    <div class="row">
+        <div class="col l3">
+            <div class="hiring-box">
+                <p class="hiring-heading">Frontend Developer</p>
+                <p class="hiring-require">Experience : 2+ years</p>
+                <p class="hiring-require">Skills : BackboneJs, ReactJs, Redux, React</p>
+                <p class="hiring-button">Apply</p>
+            </div>
+        </div>
+        <div class="col l3">
+            <div class="hiring-box">
+                <p class="hiring-heading">UI/UX Designer</p>
+                <p class="hiring-require">Experience : 2+ years</p>
+                <p class="hiring-require">Skills : Sketch, PhotoShop, Illustratore, HTML/CSS</p>
+                <p class="hiring-button">Apply</p>
+            </div>
+        </div>
+        <div class="col l3">
+            <div class="hiring-box">
+                <p class="hiring-heading">Backend Developer</p>
+                <p class="hiring-require">Experience : 2+ years</p>
+                <p class="hiring-require">Skills : Python, DJango, Postgresql, DRF</p>
+                <p class="hiring-button">Apply</p>
+            </div>
+        </div>
+        <div class="col l3">
+            <div class="hiring-box">
+                <p class="hiring-heading">Data Engineer</p>
+                <p class="hiring-require">Experience : 2+ years</p>
+                <p class="hiring-require">Skills : Python, MySQL, Data Analytics, Django</p>
+                <p class="hiring-button">Apply</p>
+            </div>
+        </div>
+    </div>
+    <p class="join-us-cv">Looking for a different role?</br>
+    Drop us a message with your CV at <a href="contact.html" class="contact-link">here</a> and we will get back to you.</p>
+</section>
+
+<footer class="footer">
+  <img src="img/waves.png" class="wave-image">
+  <div class="footer-content">
+    <div class="row">
+      <div class="col l2 m2 s2">
```

Comment:
@priyanka0901 Grid should always go from small screen to big screen not from big to small. 
`For Ex: <div class="col s2 m2 l2"></div>`
If the grid value is same for all the screen then we don't have mention all the screen it should only `col s2`.